### ISSUE TITLE
Sandbox hangs inside docker

### ISSUE THREAD

**Issue by jgallen23** [architect/architect#382]
**Describe the bug**
Trying to run `npx sandbox` in Docker (node:8-alpine) and it just hangs

**To Reproduce**
1. `docker run --rm -it -v $PWD:/src --workdir /src node:8-alpine sh`
2. `npm install`
3. `npx sandbox`
4. hangs after `Shared code copied`

**Expected behavior**
Expect sandbox to say `Started HTTP "server"....`

**Screenshots**
```
/var/work/dev/sgff # npx sandbox                                                                          
        app ⌁ sgff                                                                                        
     region ⌁ us-east-1                                                                                   
    profile ⌁ sgff                                                                                        
    version ⌁ 5.7.0                                                                                       
                                                                                                          
✓ Shared code and .arc copied into 3 Functions
```

**Desktop (please complete the following information):**
 - OS: Alpine linux

**Comment by jgallen23** [comment]
I've figured out the issue.  Seems like if the profile is set inside .arc and that profile doesn't exist, the sandbox command just hangs

**Comment by ryanblock** [comment]
Hmm, I can see that. Does it replicate outside of Docker? (I assume it would.)

**Comment by jgallen23** [comment]
Yep

**Comment by ryanblock** [comment]
Sounds related to: https://github.com/architect/architect/issues/391

**Comment by ryanblock** [comment]
Inadvertently closed too early!

However, this should now be resolved in `5.9.12`. Please do let me know if you run into further trouble!
